File: 1. Learn the Basics/1. Pattern/C++/8.cpp
/*
*********
 ******* 
  *****  
   ***   
    *
*/

#include <bits/stdc++.h>
using namespace std;

int main() {
	// your code goes here
	for(int i=0;i<5;i++){
		    //Space = i
		    for(int j=0;j<i;j++){
		        cout << " ";
		    }
		     //Star -> 2n - (2i+1)
		    for(int j=0;j<10-(2*i+1);j++){
		        cout << "*";
		    }
		     //Space
		    for(int j=0;j<i;j++){
		        cout << " ";
		    }
		    cout << endl;
	}
	return 0;
}

File: 1. Pattern/Java/8.java
/* package codechef; // don't place package name! */

/*
*********
 ******* 
  *****  
   ***   
    *

*/
import java.util.*;
import java.lang.*;
import java.io.*;


/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		for(int i=0;i<5;i++)
		{
		    //Space
		    for(int j=0;j<i;j++)
		    {
		        System.out.print(" ");
		    }
		     //Star -> 2n - (2i+1)
		    for(int j=0;j<10-(2*i+1);j++)
		    {
		        System.out.print("*");
		    }
		     //Space
		    for(int j=0;j<i;j++)
		    {
		        System.out.print(" ");
		    }
		    System.out.println();
		}
	}
}
